src/Identity/test/InMemory.Test/FunctionalTest.cs

```diff
@@ -271,7 +272,7 @@ public async Task TwoFactorRememberCookieClearedBySecurityStampChange(bool testC
         Assert.Equal(HttpStatusCode.OK, transaction5.Response.StatusCode);
 
         // Wait for validation interval
-        timeProvider.Advance(TimeSpan.FromMinutes(30));
+        timeProvider.Advance(TimeSpan.FromMinutes(30) + TimeSpan.FromMilliseconds(1));
```

Comment:
Why did this need to change?

Reply:
I think the new provider is more precise and starts at a round number. When the time was advanced 30m it was exactly start + 30m which was still not greater than the 30m limit.